melodic/distribution.yaml

```diff
@@ -2523,6 +2523,21 @@ repositories:
       url: https://bitbucket.org/AndyZe/pid.git
       version: master
     status: maintained
+  platform_automation_msgs:
+    release:
+      packages:
+      - module_comm_msgs
+      - platform_automation_msgs
+      - platform_comm_msgs
```

Comment:
These package names are not descriptive enough making it very hard to guess what they contain based on the package name.
Could you brainstorm or suggest package names that would be more descriptive as per [Package Naming conventions](http://www.ros.org/reps/rep-0144.html#package-naming)?